_15thnight/__init__.py

```diff
@@ -4,44 +4,74 @@
 from flask.ext.login import LoginManager, current_user, logout_user
 from werkzeug.datastructures import MultiDict
 
-from _15thnight import database, queue
-from _15thnight.api import (
-    account_api, alert_api, category_api, need_api, response_api, service_api,
-    user_api
-)
 from _15thnight.email import mailer
 from _15thnight.forms import csrf_protect
 from _15thnight.models import User
 from _15thnight.util import ExtensibleJSONEncoder
 
 
-app = Flask(__name__)
-
-try:
-    app.config.from_object('config')
-except:
-    app.config.from_object('configdist')
+login_manager = LoginManager()
```

Comment:
Move this line and all the lines below create_app that are using the `@app` annotation into the `create_app` function please. We will want to test that stuff eventually, and for cleanliness, it should all be in `create_app`.